### PR TITLE
Project Review

### PR DESCRIPTION
**Pull request by clin2on3mun**
portfolio project: a project built with HTML and CSS 

- it is a mobile version of the portfolio
- it uses HTML semantics tags and CSS layouts.

### CODE REVIEW

**Comment by clin2on3mun** [comment]
Hi clin2on3mun , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉


## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @clin2on3mun 👏 👏 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

## Highlights
Good use of summary template ✔️ 

## Required Changes ♻️

- [ ] **Please as stated in the requirement, kindly ensure to use the** [GitHub inline comments feature](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) **to add at least 2 to review comments.** **Make sure to leave the comment under a line or block of code that fits the context the comment is about.** 

Untitled


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+                    <div class="button-section">
+                       <ul class="buttons">
+
+                           <li><button class="btn-work">CSS</button></li>
+                           <li><button class="btn-work">HTML</button></li>
+                           <li><button class="btn-work">Bootstrap</button></li>
+                           <li><button class="btn-work">Ruby</button></li>
+                           <li><button class="view-button">See project</button></li>
+                        </ul>   
+                    </div>
```
Comment (clin2on3mun) [inline index.html:63]: i would suggest you use the HTML semantics tag (eg: section) where you use div you can read about  <a href="https://www.pluralsight.com/guides/semantic-html">HTML semantics</a> to get an idea on changes requested and why you will make them

File: index.html [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+        <link rel="stylesheet" href="styles.css">
+        <title>Munana Clinton </title>
+     </head>
+     <body>
+        <header id="header">
+            <nav class="header-nav">
+                <a id="logo">Munana Clinton</a>
+                <div class="menu-bar">
+                    <span class="nav-bar">
+                    <span class="nav-bar"></span>
```
Comment (clin2on3mun) [inline index.html:18]: I would suggest you always close HTML Tags because it show that is closing tag and end of that element

**Review by clin2on3mun — COMMENTED** [review]
Hi @clin2on3mun , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


soclose

## Highlights
✅ Good GitHub Flow
✅ Great design 👏 
✅ No Linter errors

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by nahnahdev** [comment]
## Project Status: APPROVED :green_circle: 
Hi @clin2on3mun, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Inline comments added :100: 

## Optional suggestions
- Great job adding inline comments as mentioned by the previous reviewer :+1:. I will suggest that you include emojis in your reviews so it sends a friendly tone to the recipient. Also, I will suggest that you make sure your sentences are well phrased so the student understands what is expected of them and is able to deliver :100: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+                </p>  
+                <div class="my-socials">
+                    <a href="https://github.com/clin2on3mun"><img src="images/github.png" alt="github"></a>
+                    <a href="https://www.linkedin.com/in/munana-clinton-b6122720b/"><img src="images/linkedIn.png" alt="linkedIn"></a>
+                    <a><img src="images/angelist.png" alt="Angelist"></a>
+                    <a href="https://twitter.com/ClintonMunana"><img src="images/twitter.png" alt="twitter"></a>
+                    <a><img src="images/Medium.png" alt="Medium"></a>
+                         
+                </div>
+            </div>
```
Comment (uno36) [inline index.html:42]: Great Job on the work so far you have put much effort, 👦

- I suggest using a list for your links this makes it easier for you to target them in your html for example
~~~~
 <ul>
          <li> link1 <li>
          <li> link2 <li>
           <li> link3 <li>
            <li> link4 <li>
 <ul>
~~~~

**Pull request merged**